kitsune/notifications/api.py

```diff
@@ -27,7 +29,7 @@ def has_object_permission(self, request, view, obj):
 
 
 class NotificationSerializer(serializers.ModelSerializer):
-    is_read = serializers.Field(source='is_read')
+    is_read = serializers.ReadOnlyField()
```

Comment:
In DRF 2.x, serializers.Field was read-only, and serializers.WriteField was the writable one. In DRF 3.x, serializers.ReadOnlyField is read-only, and serializers.Field tries to be a writable field by default.